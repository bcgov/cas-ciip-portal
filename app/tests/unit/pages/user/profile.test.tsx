import React from 'react';
import {shallow} from 'enzyme';
import Profile from 'pages/user/profile';
const query = {
  user: {
    id: 'WyJ1c2VycyIsMV0='
  }
};
describe('user profile', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(<Profile query={query} />);
    expect(wrapper).toMatchSnapshot();
  });
});
