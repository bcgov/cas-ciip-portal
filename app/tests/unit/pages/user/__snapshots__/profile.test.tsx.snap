// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`user profile matches snapshot 1`] = `
<Relay(DefaultLayout)>
  <Relay(UserProfile)
    user={null}
  />
</Relay(DefaultLayout)>
`;
